source/core/TDCodecVia.cpp

```diff
@@ -1074,6 +1074,21 @@ Boolean TDViaParser::EatJSONItem(SubString* input)
     return true;
```

Comment:
@shivaprasad-basavaraj Our PR workflow also includes making sure the [squad workflow tests](https://ni.visualstudio.com/DevCentral/_wiki/wikis/AppCentral.wiki/2445/How-to-update-Vireo-in-NXG?anchor=trigger-the-web-module-squad-tests) are run with the [local changes](https://github.com/ni/VireoSDK/blob/main/docs/Workflow.md#testing-vireojs-on-a-different-machine--ci) and including a link to the CI build.